### DISCUSSION TITLE
Quills Emoji In Nextjs

### DISCUSSION THREAD

**Discussion by someskumar-k** [vercel/next.js#11979] [Help]
Hi I had implemented quill text editor using dynamic 
```
const QuillNoSSRWrapper = dynamic(
    import('react-quill'), { ssr: false, loading: () => <p>Loading ...</p> }
)
```
The editor is working fine, but Is there any way possible to use qullis-emoji and other custom modules ?
If it possible kindy update a sample code

**Comment by satyamskillz** [comment]
I am trying to do the same thing, if you have found any solution then please share?

**Reply by someskumar-k** [reply]
I had created a custom wrapper and imported all modules then imported the custom wrapper as dynamic import @satyamskillz

**Reply by satyamskillz** [reply]
I will try this, however, I have another issue with react-quill, It is using an old version of React which is causing an error in the production. what should I do? now 
Now, I am thinking to replace the react-quill with draft.js or slate.

**Reply by ghost** [reply]
I am stuck here with this quill editor, have you resolve this emoji picker working? Please give me advices for this

**Reply by satyamskillz** [reply]
> I am stuck here with this quill editor, have you resolve this emoji picker working? Please give me advices for this 

I was unable to fix it and now use tiptap.js

**Comment by andresmv94** [comment]
> Hi I had implemented quill text editor using dynamic
> 
> ```
> const QuillNoSSRWrapper = dynamic(
>     import('react-quill'), { ssr: false, loading: () => <p>Loading ...</p> }
> )
> ```
> 
> The editor is working fine, but Is there any way possible to use qullis-emoji and other custom modules ? If it possible kindy update a sample code

Hi!! do you find the solution? Can you post an example code for Emoji In Nextjs?
